macros/fivetran_date_macros/fivetran_convert_timezone.sql

```diff
@@ -0,0 +1,41 @@
+{%- macro fivetran_convert_timezone(column, target_tz=None, source_tz=None) -%}
+{%- set source_tz = "UTC" if not source_tz else source_tz -%}
+{%- set target_tz = "UTC" if not target_tz else target_tz -%}
```

Comment:
Note the difference https://github.com/calogica/dbt-date/blob/main/macros/calendar_date/convert_timezone.sql#L3